canvas_modules/common-canvas/__tests__/common-properties/actions/image-action-test.js

```diff
@@ -96,86 +107,94 @@ describe("action-image renders correctly", () => {
 			done();
 		}
 		controller.setHandlers({ actionHandler: callback });
-		const wrapper = mount(
+		const wrapper = render(
 			<Provider store={controller.getStore()}>
 				<ActionImage
 					action={action}
 					controller={controller}
 				/>
 			</Provider>
 		);
-		const image = wrapper.find("img");
-		image.simulate("click");
+		const image = wrapper.getByRole("img");
+		fireEvent.click(image);
 	});
 	it("action image renders when disabled", () => {
 		controller.updateActionState(actionStateId, "disabled");
-		const wrapper = mount(
+		const wrapper = render(
 			<Provider store={controller.getStore()}>
 				<ActionImage
 					action={action}
 					controller={controller}
 				/>
 			</Provider>
 		);
-		const imageWrapper = wrapper.find(".properties-action-image");
-		expect(imageWrapper.hasClass("disabled")).to.equal(true);
+		const image = wrapper.getByRole("img");
+		const imageWrapper = image.parentElement.parentElement.parentElement.parentElement;
+		expect(imageWrapper.className.includes("disabled")).to.equal(true);
 	});
 	it("action image renders when hidden", () => {
 		controller.updateActionState(actionStateId, "hidden");
-		const wrapper = mount(
+		const wrapper = render(
 			<Provider store={controller.getStore()}>
 				<ActionImage
 					action={action}
 					controller={controller}
 				/>
 			</Provider>
 		);
-		const imageWrapper = wrapper.find(".properties-action-image");
-		expect(imageWrapper.hasClass("hide")).to.equal(true);
+		const image = wrapper.getByRole("img");
+		const imageWrapper = image.parentElement.parentElement.parentElement.parentElement;
```

Comment:
and this

Reply:
I fixed all the areas where I used numerous parent elements.